app/components/time_ago_component.rb

```diff
@@ -0,0 +1,41 @@
+# frozen_string_literal: true
+
+module Primer
+  # A basic wrapper for the "<time-ago>" element.
+  class TimeAgoComponent < Primer::Component
+    def initialize(time:, micro: false, **system_arguments)
+      @system_arguments = system_arguments
+      @system_arguments[:datetime] = time.utc.iso8601
+      @system_arguments[:classes] = class_names("no-wrap", @system_arguments[:classes])
+      @system_arguments[:tag] = "time-ago"
+      @system_arguments[:format] = "micro" if micro
+      @content = if micro
```

Comment:
nit: might we call this `@time`?
`@content` is an internal ivar from the view_component framework

Reply:
I've refactored this out more so the initialiser just assigns variables an does not apply logic, the logic is then in a `time_in_words` method.